tests/test_recreation.py

```diff
@@ -581,14 +581,14 @@ def test_all_metrics_local_server(self):
         expected_grid_vector_path = os.path.join(
             REGRESSION_DATA, 'predictor_data_all_metrics.shp')
         utils._assert_vectors_equal(
-            out_grid_vector_path, expected_grid_vector_path, 1e-3)
+            expected_grid_vector_path, out_grid_vector_path, 1e-3)

```

Comment:
When it fails, the message from this assertion function expects the first arg to be the "expected" value and second to be the "actual".